package.json

```diff
@@ -61,13 +61,13 @@
     "istanbul": "0.2.7",
     "jsxgettext": "0.8.2",
     "load-grunt-tasks": "0.4.0",
-    "nock": "5.0.0",
+    "nock": "8.0.0",
     "proxyquire": "0.4.1",
     "time-grunt": "0.3.1",
     "turbo-test-runner": "0.3.3"
   },
   "engines": {
-    "node": ">=0.4"
+    "node": "4.4"
```

Comment:
we should not restrict the version of node here. fhc is running on user machines and they could have different versions of node. Here we should just specify the minimum required node version. So we should keep the original requirement